appveyor.yml

```diff
@@ -45,10 +45,12 @@ install:
   - IF NOT EXIST php-installed.txt echo assert.exception=On >> php.ini
   - IF NOT EXIST php-installed.txt appveyor DownloadFile https://getcomposer.org/composer.phar
   - IF NOT EXIST php-installed.txt echo @php %%~dp0composer.phar %%* > composer.bat
+  - IF NOT EXIST php-installed.txt appveyor DownloadFile https://phar.phpunit.de/phpunit.phar
+  - IF NOT EXIST php-installed.txt echo @php %%~dp0phpunit.phar %%* > phpunit.bat
```

Comment:
is there a specific reason to not use phive on appveyor?

Reply:
That change is gone now... went back to require-dev having phpunit.